src/components/Tokens/TokenRowWallets.tsx

```diff
@@ -34,6 +162,55 @@ export const TokenRowWallets = component$<TokenRowWalletsProps>(
       resolutions: [1920, 1280],
       imageTransformer$,
     });
+
+    const currentBalanceOfToken = useSignal("");
+    const latestTokenPrice = useSignal("");
+    const latestBalanceUSD = useSignal("");
+
+    // eslint-disable-next-line qwik/no-use-visible-task
+    useVisibleTask$(async ({ cleanup }) => {
+      if (walletId === undefined) {
+        throw new Error("walletId is undefined");
+      }
+      const data = await tokenRowWalletsInfoStream(walletId, symbol);
+
+      cleanup(async () => {
```

Comment:
I would always move registration of `cleanup` to the very top of the function (this may be standard to increase code readability)

Reply:
on point, fixed